src/common/tests/SConscript

```diff
@@ -52,7 +52,7 @@ def scons():
                     LIBS=['daos_common', 'gurt', 'pmemobj', 'cmocka'])
     daos_build.test(tenv, 'other', 'other.c',
                     LIBS=['daos_common', 'gurt', 'cart'])
-    daos_build.test(tenv, 'common_test',
+    common_test = daos_build.test(tenv, 'common_test',
```

Comment:
Not really your thing but we should probably just install all of these tests.  Though internal tests should probably go somewhere like install/TESTING/utest or something

Reply:
Good idea. I'll create a ticket for it. Rather not do it now so I can get this change in.